cmd/ocm/gcp/update-wif-config.go

```diff
@@ -71,7 +78,7 @@ func updateWorkloadIdentityConfigurationCmd(cmd *cobra.Command, argv []string) e
 		return errors.Wrapf(err, "failed to initiate GCP client")
 	}
 
-	if UpdateWifConfigOpts.DryRun {
+	if UpdateWifConfigOpts.Mode == ModeManual {
```

Comment:
create and delete cmds  validate the value of Mode to be one of the two,  shouldnt there be a  similar check for update as well?